corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -148,6 +148,18 @@ <h4 class="panel-title panel-title-nolink">{% trans "Search and Claim Options" %
             <!-- /ko -->
           </div>
           {% endif %}
+          {% if request|toggle_enabled:'DATA_REGISTRY' %}
+          <div class="form-group">
+              <label for="data-registry-id" class="{% css_label_class %} control-label">
+                  {% trans "Registry ID" %}
+                  <span class="hq-help-template" data-title="{% trans "Registry ID" %}"
+                        data-content="{% trans_html_attr "Enter the ID of a case registry.  Users with access will see results from the registry when performing this search." %}">
+              </label>
+              <div class="{% css_field_class %}">
+                  <input class="form-control" type="text" id="data-registry-id" data-bind="value: dataRegistryId"/>
```

Comment:
any chance of making this a select box with the actual registry names?

Reply:
Oh yeah, not sure why I didn't do that

Reply:
Added and updated the screenshot above.  I left the display as just the name, which isn't guaranteed to be unique.  IMO that's probably fine, and would be easy to change down the line if needed.